tests/tests_twobody/test_orbit.py

```diff
@@ -52,11 +50,7 @@
 from poliastro.frames.util import get_frame
 from poliastro.twobody.angles import E_to_M, nu_to_E
 from poliastro.twobody.orbit import Orbit
-from poliastro.warnings import (
-    OrbitSamplingWarning,
-    PatchedConicsWarning,
-    TimeScaleWarning,
-)
+from poliastro.warnings import OrbitSamplingWarning, PatchedConicsWarning
```

Comment:
also not a fan of these kinds of refactoring changes as they add problems later to the git blame 😅, but the quality checks won't allow it for now.

Reply:
the PEP8 / Black deity rules 😈